content/80.glossary.md

```diff
@@ -1,3 +1,7 @@
 ## Glossary
 
-* **Term**: Write the definition as a sentence.
+| Consensus term | Synonyms | Description |
+| -------------- | -------- | ----------- |
+| Network | Graph | A set of *nodes*, representing objects of interest, linked by *edges*, representing specific relationships between nodes. Nodes can have several properties called *attributes* like cell type or position. |
+| Node | Vertice | Element of interest in a network and linked to other nodes. For exemple: people, cells, proteins or genes. |
+| Edge | Link | The relatioinship between 2 nodes in a network. For exemple: friendship in social networks, cells in contact in a spatial network, protein-protein interactions. |
```

Comment:
```suggestion
| Edge | Link | The relationship between 2 nodes in a network. For example: friendship in social networks, cells in contact in a spatial network, or gene-gene interactions in a genetic network. |
```
Some typo changes, and then I also made it gene-gene interactions (a la epistasis or similar) since otherwise I was writing physical protein-protein interactions in a protein network and it came right after the cells. See what you think!

Reply:
```suggestion
| Edge | Link | The relationship between 2 nodes in a network. For example: friendship in social networks, cells in contact in a spatial network, or gene-gene interactions in a gene regulatory network. |
```
Yes it's globally much better, I propose 'gene regulatory network' instead of 'genetic network' because some biologists prefer it this way (although I personally don't mind)